project-base/storefront/components/Pages/Order/TransportAndPayment/TransportAndPaymentSelect/TransportAndPaymentSelect.tsx

```diff
@@ -102,6 +102,24 @@ export const TransportAndPaymentSelect: FC<TransportAndPaymentSelectProps> = ({
                     />
                 }
             />
+            {paymentItem.uuid === payment?.uuid &&
+                payment.type === 'goPay' &&
+                payment.goPayPaymentMethod?.identifier === 'BANK_ACCOUNT' && (
```

Comment:
Can we have separated variable for this so JSX is be better readable?

Reply:
I also put the select in a separate variable.